mne/time_frequency/tfr.py

```diff
@@ -1511,9 +1513,28 @@ def plot_joint(self, timefreqs=None, picks=None, baseline=None,
             fmin = freq - freq_half_range
             fmax = freq + freq_half_range
 
+            data = tfr.data
+
+            if layout is None:
+                pos = find_layout(tfr.info).pos
+            else:
+                pos = layout.pos
```

Comment:
Here I would prefer: `layout = find_layout(tfr.info).pos if layout is None else layout.pos`